tests/js/server/resilience/failover-view/resilience-synchronous-repl-with-arangosearch-view-cluster-grey.js

```diff
@@ -149,22 +152,24 @@ function SynchronousReplicationWithViewSuite () {
     assertTrue(pos >= 0);
     assertTrue(suspendExternal(global.instanceInfo.arangods[pos].pid));
     console.info("Have failed leader", leader);
+    failedState.leader = leader;
     return leader;
   }
 
 ////////////////////////////////////////////////////////////////////////////////
 /// @brief heal the follower
 ////////////////////////////////////////////////////////////////////////////////
 
-  function healLeader() {
-    var leader = cinfo.shards[shards[0]][0];
+  function healLeader(leader) {
+    if (leader == null) leader = cinfo.shards[shards[0]][1];
```

Comment:
This change modified the array subset for `cinfo.shards[...][x]` from `x = 0` to `x = 1`. Is that intentional? 
```suggestion
    if (leader === null) {
      leader = cinfo.shards[shards[0]][1];
    }
```